docs/content/preview/api/ysql/exprs/geo_partitioning_helper_functions/func_yb_server_zone.md

```diff
@@ -39,6 +39,7 @@ If you didn't set the placement_region flag at node startup, yb_server_region()
 {{< /note >}}
```

Comment:
this note is about region; is there a zone equivalent, or should it just be deleted?